src/views/ConfigurationGeneralView.vue

```diff
@@ -180,10 +180,10 @@ const isValidSocketConnectionURI = (value: string): boolean | string => {
   return true
 }
 
-const resetGlobalAddress = (): void => {
+const resetGlobalAddress = async (): Promise<void> => {
   newGlobalAddress.value = defaultGlobalAddress
 
-  setGlobalAddress()
+  await setGlobalAddress()
```

Comment:
Why would this be async?

Reply:
It's defined as async
https://github.com/bluerobotics/cockpit/blob/9d9dd630a2b090b745e016d4ab1317028ebb022d/src/views/ConfigurationGeneralView.vue#L214

Reply:
Humnnnn, the form validation is async, didn't know that.